libs/sdk-ui-dashboard/src/model/commandHandlers/layout/rowHeaderBlurredHandler.ts

```diff
@@ -0,0 +1,24 @@
+// (C) 2021-2022 GoodData Corporation
+import { SagaIterator } from "redux-saga";
+import { DashboardContext } from "../../types/commonTypes";
+import { RowHeaderBlur } from "../../commands/layout";
+import { RowHeaderBlurred, rowHeaderBlurred } from "../../events/layout";
+import { put } from "redux-saga/effects";
+import { layoutActions } from "../../store/layout";
+
+export function* rowHeaderBlurredHandler(
+    ctx: DashboardContext,
+    cmd: RowHeaderBlur,
+): SagaIterator<RowHeaderBlurred> {
+    const {
+        payload: { index },
+        correlationId,
+    } = cmd;
+
+    yield put(
+        layoutActions.setActiveHeaderIndex({
```

Comment:
Why we need this command handler and commands for layout blur / focus?
I think that activeHeaderIndex would perfectly fit into the `model/store/ui` as it is only ui thing, specific only for the runtime. And then you can call relevant ui slice actions directly from the components onBlur / onFocus handlers. It would simplify your implementation a lot.